dashboard/15-final/README.md

```diff
@@ -2,4 +2,4 @@
 
 This is the final template for the Next.js App Router Course. It contains the final code for the dashboard application.
 
-For more information, see the [course curriculum]() on the Next.js Website.
+For more information, see the [course curriculum](xxx) on the Next.js Website.
```

Comment:
Should this just be `https://nextjs.org/learn`?

Reply:
Yes!

Reply:
```suggestion
For more information, see the [course curriculum](https://nextjs.org/learn) on the Next.js Website.
```